Week3/homework/step2-3.js

```diff
@@ -19,6 +22,10 @@ function repeatStringNumTimesWithWhile(str, num) {
   let result = '';
 
   // Replace this comment and the next line with your code
+  const i = 0;
+  while (i < num) {
+    result += str;
+  }
```

Comment:
This while loop runs forever. The test will never complete. Can you figure out what's wrong here?